### DISCUSSION TITLE
Practicas HTML - Página de Pastelero

### DISCUSSION THREAD

**Discussion by OmarVenturaP** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#149] [Q&A]
Que tal Explorers, buen día, fíjense que estoy analizando el caso de la Pastelería de las practicas de HTML, pero la verdad hay algo que me tiene un poco confuso, el ejercicio menciona lo siguiente:

- El pastelero necesita tener la información de la cantidad de sabores "que le quedan" para hacer los pasteles.
- El pastelero necesita tener la información de la cantidad de adornos "que le quedan" para hacer los pasteles.
- El pastelero necesita una tabla donde aparezcan los pedidos que se han realizado con los datos del formulario.

Según le entiendo, hay que hacer una página como de inventario, y a la vez un sistema donde le aparezcan los datos del pedido realizado... 

Por ello quería de su apoyo. Realmente ¿hay algún proceso que solo implique HTML para realizarlo? y de ser así, ¿Cómo se llamaría la funcionalidad? Ya busque por internet y no logro encontrar la forma o el código para dicho propósito.

Se que como dice el MC: "no es necesario que tenga funcionalidad completa en la información", pero quisiera saber si se puede... y de ser así pues probar e implementar el código en la práctica.

De antemano, muchas gracias por su ayuda.

**Comment by avexyk** [comment]
Pues tomando en cuenta que no vemos base de datos, todo debe de generarse y mantenerse en el front-end de forma estática. Y solo se restablecería la información cuando realicen un F5. La verdad no lo he leído y no he visto los lives de HTML, apenas acabo de enviar la práctica de requerimientos, así que por el momento puede que haya explicado algo al respecto tal vez. Pero yo a ciegas haría eso de mantener la información en el front-end. En ese caso sería puro JavaScript para hacerlo.

**Comment by Burstniart** [comment]
Se puede si
Cómo?
Puedes hacer con JavaScript y bases de datos, aquí ya estarías en back end, si te quieres adelantar puedes emepzar a estudiar de recursos en línea pero lo mejor sería que primero entendieras las bases, por eso apenas estamos iniciando con html.

**Reply by OmarVenturaP** [reply]
Si, eso lo entiendo, pero como es parte de las practicas, pensé que tenia algo que ver con lo que estamos aprendiendo... Pero de hecho si supuse que era algo de backend porque se trata de datos.

**Reply by avexyk** [reply]
No es necesario hacer backend. Lo puedes manejar con JavaScript y mantener el procesamiento de datos en el front-end, obviamente no existira persistencia de datos pero para hacer una demostración de como funcionaría estaría bien.

**Reply by sergiolopezloya** [reply]
No es necesario meterse con JavaScript, es maquetación, por lo que información ficticia es suficiente

**Comment by patricia2454** [comment]
Estoy atorada en el mismo lugar, ya hice la tabla con tr th y td pero como dicen arriba de forma estatica, he buscado información y solo encuentro con PHP, se me había ocurrido una grafica de barras pero la quisiera que tuviera interacción, al parecer la dejare estatica.

**Reply by hackit92** [reply]
También podrías apoyarte con algo de javascript para darle dinamismo de una forma más "ligera" que con PHP

**Reply by OmarVenturaP** [reply]
El compañero @avexyk compartió un link que me ayudó mucho. En su repo aparece.

**Comment by AngelCruzO** [comment]
Lo que pide la practica es de forma estatica, en otras palabras solo es realizar una plantilla, maquetación HTML. Para hacerla dinamica es mejor esperar a la Mision de Backend, puesto que ahi nos enseñaran todo lo relacionado a la persistencia de datos y como mostrarlos en el front

**Comment by JuanSalvador73** [comment]
Disculpen, pero hasta donde me quedé el jueves, ¿que no era un sitio de Tacos lo que se estaba llevando a cabo?

**Reply by OmarVenturaP** [reply]
Cada semana se dejan prácticas relacionado al tema que se está analizando, en esa semana es HTML, asi que compartieron ya la práctica para realizarla, te proporciono el link: 

https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/tree/main/02%20-%20HTML/practicas

**Reply by JuanSalvador73** [reply]
Mil gracias por la info. No la había visto. :D

**Reply by BrawnyGG** [reply]
@JuanSalvador73 jajaja, yo también me había quedado con los tacos y ya había hecho mi página acerca de eso... habrá que reutilizar la plantilla

**Reply by JuanSalvador73** [reply]
[BrawnyGG](https://github.com/BrawnyGG) Ni hablar, ahora tacos y pasteles :X

**Comment by Ingrzs** [comment]
Todo estático, de ser dinámico tendrías que poner bd y hacer conexiones, si tu quieres hazlo pero expandirias mas el proceso por los inventarios y los pedidos.

**Comment by linettelopb** [comment]
Si concuerdo que debe ser estatico ya que por ahora es frontend y mas si no tienen tanta experiencia con backend  es mejor esperar como mencionan los compañeros. 
El ejemplo te habla de lo que se espera que deba hacer mas no que lo tiene que hacer tal cual.
Pero si quieres ya darle dinamismo lo puede hacer con javascript como mencionan o con algun framework que facilita bastante estas interacciones sin datos (En mi trabajo esto realizo si aun no tengo la informacion creo todas  las acciones o interacciones y despues solo agrego los datos.).
